torch/csrc/dynamo/debug_macros.h

```diff
@@ -53,6 +53,25 @@ extern "C" {
 
 #endif
 
+// e.g. INSPECT(obj1, obj2)
+// in order to inspect obj1, obj2 at the Python level, in pdb.
+// Alias for torch._dynamo.utils._breakpoint_for_c_dynamo(...)
+// WARNING: makes a Python function call! Make sure eval frame callback is
+// unset!
```

Comment:
> makes a Python function call! Make sure eval frame callback is unset!

Any chance we could automate this in the macro, like adding some wrapper calls to unset and reset the callback?